appengine/flexible_python37_and_earlier/hello_world/app.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Google LLC
```

Comment:
Ditto.

Reply:
I think I fixed all of them. Please take another look.